nengo/params.py

```diff
@@ -416,7 +417,7 @@ def validate(self, instance, function_info):
         super(FunctionParam, self).validate(instance, function)
 
 
-class FrozenObject(object):
+class FrozenObject(with_metaclass(DocstringInheritor)):
```

Comment:
From what I vaguely know about metaclasses, I think that would make sense. Maybe that's why it wasn't working for me before. I definitely had it like @tbekolay did above.